setup.py

```diff
@@ -60,7 +60,7 @@
         ],
         exclude=["tests", "tests.*"],
     ),
-    package_data={"": ["*.yml", "*.html", "poc.zip"]},
+    package_data={"": ["log.config", "*.yml", "*.html", "poc.zip"]},
```

Comment:
maybe it's better to include the whole resource folder, instead of individual file?